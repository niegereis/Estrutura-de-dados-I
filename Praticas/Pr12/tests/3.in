Early novels
============

Although early forms of the novel are to be found in a number of places, including classical Rome, 10th– and 11th-century Japan, and Elizabethan England, the European novel is often said to have begun with Don Quixote in 1605.[11]

Early works of extended fictional prose, or novels, include works in Latin like the Satyricon by Petronius (c. 50 AD), and The Golden Ass by Apuleius (c. 150 AD), works in Sanskrit such as the 6th– or
7th-century Dasakumaracarita by Dandin, and in the 7th-century Kadambari by Banabhatta, the 11th-century Japanese Tale of Genji by Murasaki Shikibu, the 12th-century Hayy ibn Yaqdhan (or Philosophus
Autodidactus, the 17th-century Latin title) by Ibn Tufail, who wrote in Arabic, the 13th-century Theologus Autodidactus by Ibn al-Nafis, another Arabic novelist, and in Chinese in the 14th-century
Romance of the Three Kingdoms by Luo Guanzhong.

Murasaki Shikibu's Tale of Genji (1010) has been described as the world's first novel[12][13] and shows essentially all the qualities for which Marie de La Fayette's novel La Princesse de Cleves
(1678) has been praised: individuality of perception, an interest in character development, and psychological observation.[14] Urbanization and the spread of printed books in Song Dynasty (960-1279)
China led to the evolution of oral storytelling into consciously fictional novels by the Ming dynasty (1368–1644). Parallel European developments did not occur for centuries, and awaited the time when
the availability of paper allowed for similar opportunities.

By contrast, Ibn Tufail's Hayy ibn Yaqdhan and Ibn al-Nafis' Theologus Autodidactus are works of didactic philosophy and theology. In this sense, Hayy ibn Yaqdhan would be considered an early example
of a philosophical novel,[15][16] while Theologus Autodidactus would be considered an early theological novel.[17] Hayy ibn Yaqdhan, with its story of a human outcast surviving on an island, is also
likely to have influenced Daniel Defoe's Robinson Crusoe (1719), because the work was available in an English edition in 1711.[18]

Epic poetry exhibits some similarities with the novel, and the Western tradition of the novel reaches back into the field of verse epics, though again not in an unbroken tradition. The epics of Asia,
such as the Sumerian Epic of Gilgamesh (1300–1000 BC), and Indian epics such as the Ramayana (400 BCE and 200 CE), and Mahabharata (4th century BC) were as unknown in early modern Europe as was the
Anglo-Saxon epic of Beowulf (c.750–1000 AD), which was rediscovered in the late 18th century and early 19th century. Other non-European works, such as the Torah, the Koran, and the Bible, are full of
stories, and thus have also had a significant influence on the development of prose narratives, and therefore the novel. Then at the beginning of the 18th century, French prose translations brought
Homer's works to a wider public, who accepted them as forerunners of the novel. [note 3]

Classical Greek and Roman prose narratives [note 4] included a didactic strand, with the philosopher Plato's (c.425-c.348 BC) dialogues; a satirical dimension with Petronius' Satyricon; the incredible
stories of Lucian of Samosata; and Lucius Apuleius' proto-picaresque The Golden Ass, as well as the heroic romances of the Greeks Heliodorus and Longus. Longus is the author of the famous Greek novel,
Daphnis and Chloe (2nd century A.D.).