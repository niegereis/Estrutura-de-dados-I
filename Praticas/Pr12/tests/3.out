9
a 8
accepted 1
ad 3
again 1
al 2
all 1
allowed 1
also 2
although 1
an 6
and 23
anglo 1
another 1
apuleius 2
arabic 2
are 3
as 10
asia 1
ass 2
at 1
author 1
autodidactus 4
availability 1
available 1
awaited 1
back 1
banabhatta 1
bc 3
bce 1
be 3
because 1
been 2
beginning 1
begun 1
beowulf 1
bible 1
books 1
brought 1
by 10
c 5
ce 1
centuries 1
century 13
character 1
china 1
chinese 1
chloe 1
classical 2
cleves 1
consciously 1
considered 2
contrast 1
crusoe 1
d 1
dandin 1
daniel 1
daphnis 1
dasakumaracarita 1
de 2
defoe 1
described 1
development 2
developments 1
dialogues 1
did 1
didactic 2
dimension 1
don 1
dynasty 2
early 7
edition 1
elizabethan 1
england 1
english 1
epic 3
epics 3
essentially 1
europe 1
european 3
evolution 1
example 1
exhibits 1
extended 1
famous 1
fayette 1
fictional 2
field 1
first 1
for 3
forerunners 1
forms 1
found 1
french 1
full 1
genji 2
gilgamesh 1
golden 2
greek 2
greeks 1
guanzhong 1
had 1
has 2
have 3
hayy 4
heliodorus 1
heroic 1
homer 1
human 1
ibn 8
in 16
include 1
included 1
including 1
incredible 1
indian 1
individuality 1
influence 1
influenced 1
interest 1
into 2
is 3
island 1
its 1
japan 1
japanese 1
kadambari 1
kingdoms 1
koran 1
la 2
late 1
latin 2
led 1
like 1
likely 1
longus 2
lucian 1
lucius 1
luo 1
mahabharata 1
marie 1
ming 1
modern 1
murasaki 2
nafis 2
narratives 2
nd 1
non 1
not 2
note 2
novel 11
novelist 1
novels 3
number 1
observation 1
occur 1
of 26
often 1
on 2
opportunities 1
or 3
oral 1
other 1
outcast 1
paper 1
parallel 1
perception 1
petronius 2
philosopher 1
philosophical 1
philosophus 1
philosophy 1
picaresque 1
places 1
plato 1
poetry 1
praised 1
princesse 1
printed 1
prose 4
proto 1
psychological 1
public 1
qualities 1
quixote 1
ramayana 1
reaches 1
rediscovered 1
robinson 1
roman 1
romance 1
romances 1
rome 1
s 7
said 1
samosata 1
sanskrit 1
satirical 1
satyricon 2
saxon 1
sense 1
shikibu 2
shows 1
significant 1
similar 1
similarities 1
some 1
song 1
spread 1
stories 2
story 1
storytelling 1
strand 1
such 4
sumerian 1
surviving 1
tale 2
th 14
the 44
them 1
then 1
theological 1
theologus 3
theology 1
therefore 1
this 1
though 1
three 1
thus 1
time 1
title 1
to 5
torah 1
tradition 2
translations 1
tufail 2
unbroken 1
unknown 1
urbanization 1
verse 1
was 3
well 1
were 1
western 1
when 1
which 2
while 1
who 2
wider 1
with 5
work 1
works 6
world 1
would 2
wrote 1
yaqdhan 4
